test/jest.e2e.setup.ts

```diff
@@ -2,13 +2,16 @@
 require('ts-node/register')
 
 import { setup as setupDevServer } from 'jest-dev-server'
+import path from 'path'
+import { v4 as uuidv4 } from 'uuid'
 import jestSetup from './jest.setup'
 
 const setup = async () => {
   await jestSetup()
 
-  // TODO: Maybe put this somewhere else?
+  process.env.SKIP_LOAD_ENV = 'true'
   process.env.ENABLE_EXPERIMENTAL_SOCKETS = 'true'
+  process.env.DATABASE_URL = path.join(__dirname, '.test-data', `${uuidv4()}.sqlite`)
```

Comment:
I also added a random database url here so that the e2e tests don't use our current database.